extras/userdoc/md/documentation/connection-generator-interface.md

```diff
@@ -0,0 +1,55 @@
+# The Connection Generator Interface
+
+Since 2014 ([Djurfeldt et al., 2014](http://dx.doi.org/10.3389/fninf.2014.00043)) NEST supports the Connection Generator Interface. This allows to couple connection generating libraries to NEST without having to modify NEST itself.
+
+In contrast to the [built-in connection functions](connection-management.md) and the Topology module, the Connection Generator Interface has a different way for specifying the connectivity:
+
+2. pre- and postsynaptic neurons are created
```

Comment:
The enumeration seems garbled, shouldn't all lines begin with 1. for automatic numbering?